extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_index.php

```diff
@@ -1,72 +1,74 @@
-				<span class="wds-global-navigation__start-a-wiki-caption"><?= DesignSystemHelper::renderText( $model['create_wiki']['header']['title'] ) ?></span>
-				<?= DesignSystemHelper::getSvg(
-					'wds-icons-plus',
-					'wds-global-navigation__start-a-wiki-icon wds-icon'
-				) ?>
-			</a>
+			<form class="wds-global-navigation__search" action="<?= Sanitizer::encodeAttribute( $model['search']['module']['results']['url'] ); ?>">
+				<div class="wds-global-navigation__search-input-wrapper">
+					<label class="wds-global-navigation__search-label">
+						<?= DesignSystemHelper::getSvg(
+							'wds-icons-magnifying-glass',
+							'wds-icon wds-icon-small wds-global-navigation__search-label-icon'
+						) ?>
+						<input id="searchInput" class="wds-global-navigation__search-input"
+							name="<?= Sanitizer::encodeAttribute( $model['search']['module']['results']['param-name'] ); ?>"
+							data-active-placeholder="<?= DesignSystemHelper::renderText( $model['search']['module']['placeholder-active'] ); ?>"
+							placeholder="<?= DesignSystemHelper::renderText( $model['search']['module']['placeholder-inactive'] ); ?>"/>
+					</label>
+					<button class="wds-button wds-is-text wds-global-navigation__search-close" type="reset" data-tracking-label="global-navigation-search-close">
+						<?= DesignSystemHelper::getSvg(
+							'wds-icons-cross',
+							'wds-icon wds-icon-small wds-global-navigation__search-close-icon',
+							wfMessage( 'global-navigation-search-cancel' )->escaped()
+						) ?>
+					</button>
+				</div>
+				<button class="wds-button wds-global-navigation__search-submit" data-tracking-label="global-navigation-search-submit">
+					<?= DesignSystemHelper::getSvg(
+						'wds-icons-arrow',
+						'wds-icon wds-icon-small wds-global-navigation__search-submit-icon'
+					) ?>
+				</button>
+			</form>
+			<?php if ( isset( $model['user'] ) ): ?>
+				<?= $app->renderView(
+					'DesignSystemGlobalNavigationService',
+					'links',
+					[
+						'model' => $model['user'],
+						'type' => 'user-menu',
+						'dropdownRightAligned' => true,
+					]
+				); ?>
+				<?= $app->renderView( 'DesignSystemGlobalNavigationWallNotificationsService', 'index' ); ?>
+			<?php elseif ( isset( $model['anon'] ) ): ?>
+				<?= $app->renderView( 'DesignSystemGlobalNavigationService', 'accountNavigation', [ 'model' => $model['anon'] ] ); ?>
+			<?php endif; ?>
+			<div class="wds-global-navigation__start-a-wiki <?= isset( $model['user'] ) ? ' wds-user-is-logged-in' : ''; ?>">
+				<a href="<?= Sanitizer::encodeAttribute( $model['create_wiki']['header']['href'] ); ?>" class="wds-button wds-is-squished wds-is-secondary" data-tracking-label="<?= Sanitizer::encodeAttribute( $model['create_wiki']['header']['title']['key'] ) ?>">
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "create_wiki" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "header" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)